src/Mouf/Html/Utils/WebLibraryManager/WebLibraryManager.php

```diff
@@ -82,56 +79,92 @@ public function setWebLibraries($libraries) {
      * This function should be called within the head tag.
      *
      */
-    public function toHtml($type = null)
+    public function toHtml()
     {
         /*if ($this->rendered) {
             throw new WebLibraryException("The library has already been rendered.");
         }*/
 
+        $this->cssToHtml();
+        $this->jsToHtml();
+        $this->additionalToHtml();
+
+        $this->rendered = true;
+    }
+
+    /**
+     * Renders the HTML in charge of loading CSS files.
+     * The Html is echoed directly into the output.
+     *
+     */
+    public function cssToHtml()
```

Comment:
Sorry to bother you but could you name this function toCss() instead of cssToHtml()?

It is shorter and clearer, and since it will be part of the public API of the class, we should aim at the best possible name.